### ISSUE TITLE
Implémentation du nouveaux flux pour l'ajout une nouvelle démarche

### ISSUE THREAD

**Issue by lucaa** [DISIC/wikidemarches#1055]
Aujourd'hui l'ajout d'une nouvelle démarche est disponible à partir des outils de recherche, pour guider l'utilisateur à créer une démarche uniquement après avoir chercher des démarches et s'assurer qu'elle n'est pas déjà renseignée.

cf mockups sur figma https://www.figma.com/file/kG2znZqEzySDPUxiWbrEme/JDMA?node-id=0%3A1 , il faut changer ce flux ainsi:
* le bouton pour l'ajout d'une nouvelle démarche est disponible directement sur la page d'accueil, dans la section connectée (voir #1053 )
* l'ajout d'une nouvelle démarche ajoute automatiquement son créateur comme porteur de cette démarche (au moins dans le cas ou le créateur de la démarche à le rôle porteur - qui aujourd'hui est équivalent à être connecté sans être admin ministériel ni admin dinum)
* l'ajout d'une nouvelle démarche à partir de la page d'accueil connecté renvoie l'utilisateur sur la page d'accueil (au lieu de le laisser sur la page de la démarche), en affichant un message en haut de l'écran pour confirmer l'ajout de la démarche, comme un 
"wizard":
  * La création présente un bouton de retour à l'étape précédente (le visuel de ce formulaire **n'est pas** à implémenter dans le cadre de cette tâche):
![image](https://user-images.githubusercontent.com/657716/135505861-2c72721e-345b-429b-a578-8f4ff0d62bf5.png)
  * La création renvoie à la page d'accueil avec un message:
![image](https://user-images.githubusercontent.com/657716/135506207-f6ee56bd-11ff-412a-b346-b8963faf80da.png)

**Comment by lucaa** [comment]
@AnthonyBrunelli est-ce que c'est OK de casser cette logique: 
> Aujourd'hui l'ajout d'une nouvelle démarche est disponible à partir des outils de recherche, pour guider l'utilisateur à créer une démarche uniquement après avoir chercher des démarches et s'assurer qu'elle n'est pas déjà renseignée.

?

**Comment by AnthonyBrunelli** [comment]
@ugodtn et Kale travaillent sur les parcours, comme convenu. On fera en fonction, et on cassera la logique s'il le faut...

**Comment by AnthonyBrunelli** [comment]
@lucaa On est d'accord pour casser la logique d'obliger l'utilisateur à passer par la recherche. 

- Contraignant.
- Même si la démarche existe, elle a peut-être été faite il y a longtemps avec des infos caduques (changement ministère, changement nom, etc.).
- S’il y a des doublons, ce doit être marginal (à mesurer à l’usage), ce n’est pas bloquant et il sera toujours possible de nettoyer la base.
- Voir s’il est possible de proposer une restitution des démarches portant un nom similaire lors de la création de la démarche.

@ugodtn si tu veux compléter...

**Comment by AnthonyBrunelli** [comment]
@ugodtn (vu avec @lucaa) point structurant à trancher, cité plus haut que je remets ici : 
La création renvoie à la page d'accueil avec un message - à valider

J'imagine que c'est inclus dans les tests utilisateurs? 

Autre incertitude: où mène le bouton "Étape précédente" ? N'est-ce pas un simple bouton d'annulation ? 

Il semble aussi que dans les discussions il a été question de - à la fin du processus - activation de l'onglet "Ajouter le bouton" plutôt que d'avoir un clic pour accéder à la page. L'activation de l'onglet (et donc de rester sur la page de la démarche au lieu de revenir sur la page d'accueil) nécessite moins de développements spécifiques.

Enjeu: développements spécifiques dans XWiki.

**Comment by AnthonyBrunelli** [comment]
Question @lucaa : cette page sera certainement impactée par la 5.0 non? Pas de souci sur cet aspect? On fait d'abord une révision sur la forme/intégration du DSFR et on voit ensuite les données en elles-mêmes?

**Comment by lucaa** [comment]
> Question @lucaa : cette page sera certainement impactée par la 5.0 non? Pas de souci sur cet aspect? On fait d'abord une révision sur la forme/intégration du DSFR et on voit ensuite les données en elles-mêmes?

Oui, pas de problème sur le fait de pouvoir travailler le flux d'actions et le look & feel (si nécessaire) et de changer les éléments du formulaire après. De toute manière, l'objectif de cette tâche n'est pas de travailler le look & feel (dsfr) du formulaire, on ne risque donc pas d'investir trop dans le traitement d'un formulaire qui sera simplifié par la suite...

**Comment by lucaa** [comment]
Suite à la mise à jour des maquettes et des flux (voir la section "validée" ici https://www.figma.com/file/kG2znZqEzySDPUxiWbrEme/JDMA?node-id=90%3A24913 ), ce flux d'ajout de nouvelle page ne sera plus changé, l'utilisateur restera sur la page de la démarche après l'avoir ajoutée.

**Comment by AnthonyBrunelli** [comment]
[note: vu avec Ugo & Kale] ok ce n'est plus d'actualité on ferme.
